app/routes/__index/certificate/information.tsx

```diff
@@ -81,7 +83,11 @@ export default function CertificateInstructionsRoute() {
           </ListItem>
           <Text>
             You can check if your DNS Records have a valid certificate by using an&nbsp;
-            <Link color="brand.500" href="https://www.sslshopper.com/ssl-checker.html">
+            <Link
```

Comment:
@Eakam1007 do we need to provide colour info for links now, or did we fix this globally?  I feel like maybe you did this, but can't remember details.

Reply:
@Ririio can you change these external links to do that?  It would be good to follow the pattern from elsewhere.

Reply:
Oh I might have done adding the color attribute by instinct